Re-diffed for 2.6.14.7 by Kerin Millar <kerframil@gmail.com>:

From: Takashi Iwai <tiwai@suse.de>
Date: Thu, 5 Jan 2006 17:40:56 +0000 (+0100)
Subject: [ALSA] emu10k1 - Fix the confliction of 'Front' control
X-Git-Url: http://www.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commitdiff;h=7eae36fbd5ea9db3d3fe0d671199121be782a5b3

[ALSA] emu10k1 - Fix the confliction of 'Front' control

Modules: EMU10K1/EMU10K2 driver

Fix the confliction of 'Front' controls on models with STAC9758 codec.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---

--- linux-2.6.14.7.orig/sound/pci/emu10k1/emumixer.c	2005-10-27 19:02:08.000000000 -0500
+++ linux-2.6.14.7/sound/pci/emu10k1/emumixer.c	2006-02-06 23:24:33.000000000 -0500
@@ -750,6 +750,8 @@ int __devinit snd_emu10k1_mixer(emu10k1_
 		"Master Mono Playback Volume",
 		"PCM Out Path & Mute",
 		"Mono Output Select",
+		"Front Playback Switch",
+		"Front Playback Volume",
 		"Surround Playback Switch",
 		"Surround Playback Volume",
 		"Center Playback Switch",
